doc/working-with-the-micro-integrator.md

```diff
@@ -349,4 +349,75 @@ If you want to change the default locations of the registry directories, uncomme
     <parameter name="LocalRegRoot">{Root directory path for local Registry}</parameter>
     -->
 </registry>
-```
\ No newline at end of file
+```
+
+## Configuring and Monitoring statistics using Prometheus
+
+This Guide includes the following subsections.
+
+- [Introduction to Prometheus](#introduction-to-Prometheus)
+  - [Prometheus Publisher Component](#prometheus-Publisher-Component)
+- [Configuring Prometheus with Micro Integrator](#configuring-Prometheus-with-Micro-Integrator)
+  - [Configuring Prometheus Server](#configuring-Prometheus-Server)
+  - [Configuring Micro Integrator to Publish](#Configuring-Micro-Integrator-to-Publish)
+- [Monitoring Using Prometheus](#monitoring-Using-Prometheus)
+
+### Introduction to Prometheus
+
+[Prometheus](https://prometheus.io/) is an open source toolkit that can monitor systems and produce useful information such as graphs and 
+alerts. It collects statistical data exposed over an HTTP endpoint in the form of multi dimensional time series data,
+ which can then be visualized and queried. Since its inception, Prometheus has become a widely used monitoring tool among various companies and organizations, as it can be easily incorporated into existing systems, and is also integrable with popular graphical tools such as Grafana to visualize system behavior. Furthermore, Prometheus has its own querying language ‘PromQL’, and an expression browser to view basic graphical representations of data.
+
+The main objective of this feature is to facilitate users to make use of an extensively used system monitoring tool 
+to analyze the behavior of WSO2 Micro Integrator.  
+
+![prometheus_intro](images/prometheus_intro.png)
+
+
+#### Prometheus Publisher Component
```

Comment:
Will this section be useful for an enduser? Shall we remove this since this is some internal implementation details?

Reply:
removed in https://github.com/wso2/micro-integrator/pull/146/commits/34302a85e654ed95c64f04695cf2e99e767f2ff9